api/envoy/config/listener/v3/quic_config.proto

```diff
@@ -24,11 +24,19 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: QUIC listener config]
 
 // Configuration specific to the UDP QUIC listener.
-// [#next-free-field: 12]
+// [#next-free-field: 13]
 message QuicProtocolOptions {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.api.v2.listener.QuicProtocolOptions";
 
+  // Configuration for UDP cmsg to pass to listener filters via QuicReceivedPacket.
+  // Both level and type must be specified for cmsg to be saved.
+  message SaveCmsgConfig {
+    optional int32 level = 1;
```

Comment:
Please remove `optional` from both fields.
If the default value should be something other than 0, please use a WKT field instead.

Reply:
Done, updated to the Int32Value wrapper.